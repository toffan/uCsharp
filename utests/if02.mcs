int main() {
    if (0) {
        // nothing
    } else {
        // nothing
    }

    return 0;
}
