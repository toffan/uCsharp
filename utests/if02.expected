JUMP __main
_main:
    LOADL 0
    SUBR I2B
    JUMPIF (0) ELSE_0
    JUMP ENDIF_0
    ELSE_0:
    ENDIF_0:

    LOADL 0
    RETURN (1) 0
    RETURN (0) 0
__main:
    CALL (SB) _main
    HALT
